src/SessionManager.ts

```diff
@@ -41,6 +44,19 @@ export class SessionManager extends EventEmitter {
   private isInitialized = false;
   private handledIncomingRedirect = false;
 
+  /**
+   * Constructor for the SessionManager object. It is typically used as follows:
+   *
+   * ```typescript
+   * import { SessionManager } from "@inrupt/solid-client-authn-browser";
+   * import customStorage from "./myCustomStorage";
+   *
+   * const sessionManager = new SessionManager({
+   *   secureStorage: customStorage
+   * });
+   * ```
```

Comment:
```suggestion
   * ```
   * See {@link IStorage} for more information on how to define your own storage mechanism.
```

Assuming that initialising some `customStorage` is required? If not, a note about when to use it could be useful.

Reply:
It currently is required, but that constraint may change once the storage happens securely. Unfortunately, until I tackle this issue, it is hard to tell how the API will be impacted.